homeassistant/components/switch/amcrest.py

```diff
@@ -1,6 +1,10 @@
 """
 Support for toggling Amcrest IP camera settings.
 
+Latest change may change switch entity_id.
+Please check your automation and update the identity-id if you have previously
+assigned a name to your camera as it appends
+the name of the camera to the switch id.
```

Comment:
This information should be moved to a breaking PR section in the description of this GitHub PR

Reply:
Ok Done

Reply:
By moving he meant also removing it here in the file header.

It's only needed in the PR description. Home Assistant doesn't document code changes in the source files.

Reply:
Ok. This was easy. Thank you.